src/buildstream/_cas/casremote.py

```diff
@@ -13,6 +13,7 @@
 #
 
 import grpc
+from google.protobuf.duration_pb2 import Duration
```

Comment:
I don't know if it's fatal, but linter is complaining about this. Maybe we need to add some dependency?

Reply:
I tried to suppress this. Remains to be seen if it was successful.

Reply:
Now it should be correct suppression no-name-in-module